extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -158,21 +158,21 @@ private function reset() {
 	 * @return AchConfig instance
 	 */
 	static public function getInstance() {
-		if (!isset(self::$mInstance))
+		if( !isset( self::$mInstance ) )
 			self::$mInstance = new self();
 
 		return self::$mInstance;
 	}
 
-	private function fetchAll($useMasterDb = false) {
-		wfProfileIn(__METHOD__);
+	private function fetchAll( $useMasterDb = false ) {
+		wfProfileIn( __METHOD__ );
 
-		if(!$this->mAllDataFetched) {
-			$dbr = wfGetDB(($useMasterDb) ? DB_MASTER : DB_SLAVE);
+		if( !$this->mAllDataFetched ) {
+			$dbr = wfGetDB( ( $useMasterDb ) ? DB_MASTER : DB_SLAVE );
 			$where = array();
-			$res = $dbr->select('ach_custom_badges', 'id, type, enabled, cat, sponsored, badge_tracking_url, hover_tracking_url, click_tracking_url', $where, __METHOD__);
+			$res = $dbr->select( 'ach_custom_badges', 'id, type, enabled, cat, sponsored, badge_tracking_url, hover_tracking_url, click_tracking_url', $where, __METHOD__ );
 
-			while($row = $dbr->fetchObject($res)) {
+			while( $row = $dbr->fetchObject( $res ) ) {
 				//WARNING: if DB schema changes array will change too, it's efficient but can turn to being dangerous
 				/*
 				$store = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)